lib/claide/command/plugins_helper.rb

```diff
@@ -14,13 +14,17 @@ module PluginsHelper
       PLUGINS_RAW_URL = 'https://raw.githubusercontent.com/' \
```

Comment:
Just in specs. I'll take that out and update the specs to work off of the default `PluginsConfig`